packages/block-editor/src/components/inserter/media-tab/hooks.js

```diff
@@ -156,7 +149,15 @@ export function useMediaCategories( rootClientId ) {
 						if ( category.isExternalResource ) {
 							return [ category.name, true ];
 						}
-						const results = await category.fetch( { per_page: 1 } );
+						let results = [];
+						try {
+							results = await category.fetch( {
+								per_page: 1,
+							} );
+						} catch ( e ) {
+							// If the request fails, we shallow the error and just don't show
+							// the category, in order to not break the media tab.
+						}
```

Comment:
I see there's an async function here and a "setCategories" (setState) call at the end, meaning that potentially we could be calling setState once the component is unmounted which causes a React warning. (It's not introduced in this PR)

Reply:
This warning [has been removed in React](https://github.com/reactwg/react-18/discussions/82) now and it was mostly about `subscriptions`, that is not the case here.